tests/e2e/env-injection/01-assert-env-vars.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: java-latest-version
+  namespace: default
+spec:
+  containers:
+    - image: public.ecr.aws/odigos/java-latest-version:v0.0.1
+      name: java-latest-version
+      (env[?name=='JAVA_OPTS']):
+       - value: "-javaagent:/var/odigos/java/javaagent.jar"
+      (env[?name=='JAVA_TOOL_OPTIONS']):        
+       - value: "-javaagent:/var/odigos/java/javaagent.jar"
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: java-supported-manifest-env
+  namespace: default
+spec:
+  containers:
+    - image: public.ecr.aws/odigos/java-supported-manifest-env:v0.0.1
+      name: java-supported-manifest-env
+      (env[?name=='JAVA_OPTS']):
+      - value: "-Dnot.work=true -javaagent:/var/odigos/java/javaagent.jar"
+      (env[?name=='JAVA_TOOL_OPTIONS']): []
```

Comment:
nit: comment here why this is the expected output (as it's not trivial without all the context)